### PR TITLE
Use stepheight from CAO instead of hardcoded value

### PR DESCRIPTION
**Pull request by Ekdohibs**
Rebased #2056.
Untested however.

### CODE REVIEW

File: src/constants.h [hunk 1]
```diff
@@ -90,6 +90,12 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 // Maximum hit points of a player
 #define PLAYER_MAX_HP 20
 
+// Player weight
+#define PLAYER_DEFAULT_WEIGHT 75
+
+// Player step height
+#define PLAYER_DEFAULT_STEPHEIGHT 0.6
```
Comment (nerzhul) [inline src/constants.h]: missing f

**Comment by paramat** [comment]
> 19:34 nerzhul strangely the PR is not in good form, but idea is good

The original PR and this PR were untested, this turned out to be very broken.

**Comment by paramat** [comment]
`inline f32 getStepheight() const` oops camelcase.

**Comment by paramat** [comment]
Re-applied in #6217

**Comment by rubenwardy** [comment]
https://github.com/minetest/minetest/commit/3d0e8a691ff97a148f8c106bc23875df02226f9c

**Pull request closed without merging**